src/pages/content-edit/nav.js

```diff
@@ -53,6 +53,17 @@ module.exports = {
         });
 
         // Event handlers
+        ctrl.add = function() {
+            var result;
+
+            result = db.child("content/" + ctrl.schema.key).push({
```

Comment:
I really need to build a simple data-layer abstraction, this sort of thing living in controllers seems SO dangerous :(